CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
 **ENHANCEMENTS**
 - Echo chef-client log when a node fails to bootstrap. This helps with investigating bootstrap failures in cases CloudWatch logs are not available.
+- Add support for p6e-gb200 instances via capacity blocks.
```

Comment:
[minor] Move this change to the top of enhancements as it is the major feature in this release